packages/mds-event-server/index.ts

```diff
@@ -39,7 +39,8 @@ export const EventServer = <TData, TResult>(
       return res.status(200).send({ result })
     } catch (error) /* istanbul ignore next */ {
       await logger.error('Cloud Event', error, { method, headers, body })
-      return res.status(500).send({ error })
```

Comment:
There are basically two operations that can fail. 1) `parseCloudEvent` and 2) `await processor(...)` ... I wonder if the former should still be a `500` error since it implies that the cloud event itself is malformed.